src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -779,6 +779,10 @@ pub struct SharedFsInfo {
     #[serde(default)]
     pub virtio_fs_cache_size: u32,
 
+    /// Default size of virtqueues
+    #[serde(default)]
+    pub virtio_fs_queue_size: u32,
```

Comment:
This is unused?

Reply:
This is not used for Dragonball currently, but this file may be used for other hypervisors.